### PR TITLE
/help: Refactor description to fields and fix typos

### PR DESCRIPTION
**Pull request by Asartea**
## Because
- there is a stray bracket in the FAQ text

## This PR
- removes it

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Callbacks command: Update verbiage`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR

### CODE REVIEW

**Review by MaoShizhong — COMMENTED** [review]
Thanks @Asartea. I didn't realise this embed used a single normal string as a description. There are a couple of rogue spaces following line breaks too, which the desktop app seems to be happy trimming when rendering, but the mobile app makes it painfully obvious.
Screenshot_2024-07-02-20-30-15-41_572064f74bd5f9fa804b05334aa4f912

Would you be willing to convert the description to use the `.addFields` method instead? That should make the embed far easier to manage and prevent typos.
(edit: I did *not* expect that screenshot to render so large...)

**Comment by Asartea** [comment]
Although (possible follow up), should we import the relevant ID's from `config.js` instead and interpolate them in place?

File: new-era-commands/slash/help.js [hunk 1]
```diff
@@ -15,8 +15,12 @@ module.exports = {
...
+        { name: '**Just the FAQs**', value: 'We have a list of frequently asked questions - <#823266307293839401>!' },
+        { name: '**Helpful bot commands**', value: 'Type `/` in the message box to see the full list of commands! Explore them in our bot channel - <#513125912070455296>.' },
+        { name: '**It\'s TOP o\'Clock, 24/7/365**', value: 'This server is for supporting learners working through our curriculum.\nIf your question is not about TOP\'s curriculum or the projects it contains, try another discord: [Discord Server Search](https://disboard.org/servers/tag/coding).' },
+        { name: '**TOP is community-driven**', value: 'The Odin Project is open source and independent. Donate here: [Open Collective](https://opencollective.com/theodinproject). To learn more, use the `/contribute` or `/support` commands.' }
```
Comment (MaoShizhong) [inline new-era-commands/slash/help.js:23]:
```suggestion
        { name: 'Server Conduct', value: 'All conduct must be within our [rules](https://www.theodinproject.com/guides/community/rules), and [community expectations](https://www.theodinproject.com/guides/community/expectations).\nPlease report issues/misconduct to <@575252669443211264>, as well as to ask any rule-related questions. Modmail How-To - <#1059513837197459547>' },
        { name: 'Just the FAQs', value: 'We have a list of frequently asked questions - <#823266307293839401>!' },
        { name: 'Helpful bot commands', value: 'Type `/` in the message box to see the full list of commands! Explore them in our bot channel - <#513125912070455296>.' },
        { name: 'It\'s TOP o\'Clock, 24/7/365', value: 'This server is for supporting learners working through our curriculum.\nIf your question is not about TOP\'s curriculum or the projects it contains, try another discord: [Discord Server Search](https://disboard.org/servers/tag/coding).' },
        { name: 'TOP is community-driven', value: 'The Odin Project is open source and independent. Donate here: [Open Collective](https://opencollective.com/theodinproject). To learn more, use the `/contribute` or `/support` commands.' }
```

Nit: The `name` property already renders as bold, so `**` makes it extra-bold which looks a little jarring IMO:

image

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
> Although (possible follow up), should we import the relevant ID's from config.js instead and interpolate them in place?

This sounds very reasonable.

Also another nit, as of #554 we now officially have a `.prettierc` if you could format this file with its rules? (edit: just realised I forgot to add a format script to `package.json` in that PR... After this one, would you mind opening a new PR adding an appropriate Prettier format script to `package.json` please?)

File: new-era-commands/slash/help.js (hunk 1 above)
Comment (Asartea) [reply new-era-commands/slash/help.js:23]: oops, I thought I already fixed that but it must have gotten lost somewhere along the way

**Comment by Asartea** [comment]
> Also another nit, as of https://github.com/TheOdinProject/odin-bot-v2/pull/554 we now officially have a .prettierrc if you could format this file with its rules

Oops, completely forgot that I saw that happen earlier today

> After this one, would you mind opening a new PR adding an appropriate Prettier format script to package.json please

Sure

> This sounds very reasonable.
Going to split this out to a follow up for three reasons:
1. This also requires me to touch config.js, which quickly gets this spiraling out of scope
2. This is IMO better as a more general issue; this is not the only file suffering from hardcoded ID's
3. Its nearly 10 pm, and I'm not mentally active enough to go messing with a lot of files

**Review by MaoShizhong — APPROVED** [review]
The above is also very reasonable. Rest up well and the other stuff is just refactoring that can be done whenever. The rest of this all looks good and works well on my end too. Thank you as always!

**Pull request merged**